cdap-ui/app/cdap/components/HeaderSidebar/index.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, {PropTypes} from 'react';
+
+export default function HeaderSidebar ({onClickHandlerNoOp, getAbsUrl}) {
+  return (
+    <div className="sidebar" onClick={onClickHandlerNoOp}>
+      <a href="/"
+         className="brand sidebar-item top">
+        <div className="brand-icon text-center cdap">
+          <span className="icon-fist"></span>
+        </div>
+        {/* This will change once we introduce navbar for hydraotr & tracker in react*/}
+        <div className="product-name">
+          <span>CDAP</span>
+        </div>
+      </a>
+      <h5>Extensions:</h5>
+      <a href={getAbsUrl('hydrator')}
```

Comment:
I guess we could but its just an overhead. Essentially the `<a/>` tag here is the html component that we are using to have the appropriate url. I guess we could control the usage of `getAbsUrl` I guess. Will see what I can do.